src/__tests__/apis/responses/getUser.json

```diff
@@ -1,86 +1,100 @@
-                "expireAt": 1613831075
-            },
-            "linked_to": []
-        },
-        {
-            "id": "idn_maybe",
-            "phoneNumber": "+19515551002",
-            "reserved_for_second_factor": false,
-            "verification": {
-                "status": "verified",
-                "strategy": "phone_code",
-                "external_verification_redirect_url": null,
-                "attempts": 1,
-                "expireAt": 1613831855
-            },
-            "linked_to": []
-        }
-    ],
-    "external_accounts": [
-        {
-            "id": "idn_altavista",
-            "googleId": "1234567890",
-            "approvedScopes": "email https://www.googleapis.com/auth/userinfo.email https://www.googleapis.com/auth/userinfo.profile openid profile",
-            "emailAddress": "kyle@tech.com",
-            "givenName": "Kyle",
-            "familyName": "Reese",
-            "picture": "https://lh3.googleusercontent.com/-MvTefx9nU6o/BBBBBBBBBBI/BBBBBBBBBBB/ereRAMWiew8hQ5IaSVdTZWm1Y1G1z0T_P7Q/s96-c/photo.jpg"
+          "type": "oauth_google",
+          "id": "idn_t800"
         }
-    ],
-    "public_metadata": {},
-    "private_metadata": {},
-    "created_at": 1611948436,
-    "updated_at": 1611948436
+      ]
+    },
+    {
+      "id": "idn_invision",
+      "email_address": "john@tech.com",
+      "verification": {
+        "status": "verified",
+        "strategy": "email_code",
+        "external_verification_redirect_url": null,
+        "attempts": 1,
+        "expire_at": 1613830889
+      },
+      "linked_to": []
+    }
+  ],
+  "phone_numbers": [
+    {
+      "id": "idn_callme",
+      "phone_number": "+19515551001",
+      "reserved_for_second_factor": true,
+      "verification": {
+        "status": "verified",
+        "strategy": "phone_code",
+        "external_verification_redirect_url": null,
+        "attempts": 1,
+        "expireAt": 1613831075
+      },
+      "linked_to": []
+    },
+    {
+      "id": "idn_maybe",
+      "phoneNumber": "+19515551002",
+      "reserved_for_second_factor": false,
+      "verification": {
+        "status": "verified",
+        "strategy": "phone_code",
+        "external_verification_redirect_url": null,
+        "attempts": 1,
+        "expireAt": 1613831855
+      },
+      "linked_to": []
+    }
+  ],
+  "external_accounts": [
+    {
+      "object": "google_account",
+      "id": "gac_googlyeyes",
+      "google_id": "1234567890",
+      "approved_scopes": "email https://www.googleapis.com/auth/userinfo.email https://www.googleapis.com/auth/userinfo.profile openid profile",
+      "email_address": "kyle@tech.com",
+      "given_name": "Kyle",
+      "family_name": "Reese",
+      "picture": "https://lh3.googleusercontent.com/-MvTefx9nU6o/BBBBBBBBBBI/BBBBBBBBBBB/ereRAMWiew8hQ5IaSVdTZWm1Y1G1z0T_P7Q/s96-c/photo.jpg"
+    },
+    {
+      "object": "facebook_account",
+      "id": "fbac_xavier",
+      "facebookId": "9876543210",
```

Comment:
I notice an inconsistency here: `google_id` vs `facebookId`. It's not present in `getUserList.json` though. What's the right name?

Reply:
You're right, it's snake case, will fix.